File: app/src/main/java/org/odddev/fantlab/auth/login/ILoginView.java
package org.odddev.fantlab.auth.login;

import com.arellomobile.mvp.MvpView;

/**
 * @author kenrube
 * @since 30.08.16
 */

interface ILoginView extends MvpView {

    void showLoginResult(boolean loggedIn);

    void showError(String error);

    void showFieldsInvalid();
}

File: app/src/main/java/org/odddev/fantlab/auth/login/ILoginView.kt
package org.odddev.fantlab.auth.login

import com.arellomobile.mvp.MvpView

/**
 * @author kenrube
 * *
 * @since 30.08.16
 */

interface ILoginView : MvpView {

	fun showLoginResult(loggedIn: Boolean)

	fun showError(error: String)

	fun showFieldsInvalid()
}
